package.json

```diff
@@ -4,7 +4,11 @@
   "description": "A full Capitalisk node implementation",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postgres:startdb": "docker run --name capitalisk_main --restart always -p 5432:5432 -e POSTGRES_PASSWORD=ldpos -e POSTGRES_USER=ldpos -d postgres",
+    "postgres:createdb": "createdb capitalisk_main -h 172.17.0.2 -U ldpos",
```

Comment:
Nice. Could we make a separate set of commands for testnet too? Like `postgres:testnet:startdb` and  `postgres:testnet:createdb` (different convention is OK too)